presto-docs/src/main/sphinx/connector/thrift.rst

```diff
@@ -0,0 +1,87 @@
+================
+Thrift Connector
+================
+
+Thrift Connector makes it possible to integrate with external storage systems without a custom Presto connector implementation.
```

Comment:
These lines are a bit long. We try to keep text wrapped at 80-100 chars in the docs.